utils/ci/script.sh

```diff
@@ -0,0 +1,28 @@
+# Derived from https://github.com/japaric/trust
+
+set -ex
+
+main() {
+    if [ ! -z $DISABLE_TESTS ]; then
+        if [ ! -z $DISABLE_STD ]; then
+            cross build --all --no-default-features --target $TARGET
+        else
+            cross build --all --features log,serde-1 --target $TARGET
+        fi
+        return
+    fi
+
+    if [ ! -z $NIGHTLY ]; then
+        cross test --all --tests --no-default-features --features alloc --target $TARGET
+        cross test --all --features serde-1,log,nightly --target $TARGET
+        cross test --all --benches --target $TARGET
+    else
+        cross test --all --tests --no-default-features --target $TARGET
+        cross test --all --features serde-1,log --target $TARGET
```

Comment:
Is/should this list be the same as in `.travis.yml`?

Reply:
I don't see why that's required; e.g. I don't see much point testing docs on multiple platforms.

Reply:
To be honest I did not check the list completely. And you are right, not everything is interesting to test on all platforms. The ` --all` flag is mostly there for workspaces, which we don't use yet, right? But I see no problem in keeping it.

Reply:
I revised the tests for non-cross builds and combined the two nightly targets we've had for ages, as well as a bit more clean-up. Yes, `--all` does nothing helpful now. But I'm not sure it will do quite the right thing in the future either because I don't believe passed `--features` apply to sub-targets.